src/macro/macroApi.ml

```diff
@@ -403,6 +414,97 @@ and encode_display_mode dm =
 	in
 	encode_enum ~pos:None IDisplayMode tag pl
 
+(** encoded to haxe.display.Display.Platform, an enum abstract of String *)
+and encode_platform p =
```

Comment:
You can use existing Globals.platform_name

Reply:
Oh, nice catch! thank you! 😄